specs/SPEC-d5e56259/contracts/rest-api.yaml

```
@@ -0,0 +1,764 @@
+        defaultArgs:
+          type: array
+          items:
+            type: string
+          nullable: true
+          example: ['--verbose']
+        env:
+          type: object
+          additionalProperties:
+            type: string
+          nullable: true
+          example: { 'DEBUG': 'true' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)